Code/Editor/EditorPreferencesPageViewportGizmo.h

```diff
@@ -54,6 +54,7 @@ class CEditorPreferencesPage_ViewportManipulator : public IPreferencesPage
         AZ_TYPE_INFO(Manipulators, "{2974439C-4839-41F6-B526-F317999B9DB9}")
 
         float m_manipulatorLineBoundWidth;
+        float m_manipulatorCircleBoundWidth;
```

Comment:
Both float members could be set to 0.0f to avoid potentially be left uninitialized.

Reply:
In practice I don't think that will ever happen as we assign to them in `InitializeSettings` but I can add the initializer for extra safety

Reply:
I'm inclined to add them as it'll make it future proof in case the implementation changes for some reason in the future. But it's not big deal, it can be done in a subsequent PR.